modules/ROOT/pages/digger/cleaning-up-client-builds.adoc

```diff
@@ -0,0 +1,17 @@
+[[cleaning-up-mobile-client-builds]]
+= Clean up a Mobile Client Build
+
+== Overview
+
+This document outlines how to clean up a mobile client buildconfig and builds so that you can fully clean up the related builds and Jobs from the Mobile CI/CD service. 
+
+== Deleting an individual build
+
+After creating and running a build, you should be able to click through to your mobile client screen from the project overview and into the builds tab. In this tab you should see the details of your various mobile client builds.
+
+From here you will see the build number which is also a link to the individual build. Clicking on the build link will bring you to the build resource screen. From this screen you can select the actions and click the delete option. Once deleted the build resource will be removed from OpenShift along with the corrosponding build in the CI CD service and any artefacts created.
```

Comment:
Just an opinion, but this paragraph reads kinda clunky. This may be one of those cases where some screenshots may help?